content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -105,6 +105,10 @@ to use the last chunk of container log output if the termination message file
 is empty and the container exited with an error. The log output is limited to
 2048 bytes or 80 lines, whichever is smaller.
 
+{{< note >}}
+Message written is intended to be brief final status, such as an assertion failure message. Will be truncated by the node if greater than 4096 bytes. The total message length across all containers will be limited to 12kb. Defaults to /dev/termination-log. Cannot be updated.
```

Comment:
The kubelet truncates the message, right?

How about this text:
```suggestion
The termination message is intended to be brief final status, such as an assertion failure message. The kubelet truncates messages that are longer than 4096 bytes. The total message length across all containers will be limited to 12KiB. The default termination message path is `/dev/termination-log`. You cannot set the termination message path after a Pod is launched.
```

Also, because this is quite a lot of detail, I wouldn't put it inside a note shortcode.

Reply:
> The kubelet truncates the message, right?

From the code, this is handled in kubelet

https://github.com/kubernetes/kubernetes/blob/4e8b56e6671893757d40e2001a3c615acebc13a2/pkg/kubelet/kuberuntime/kuberuntime_container.go#L409-L429

Reply:
> Also, because this is quite a lot of detail, I wouldn't put it inside a note shortcode.

updated, thx